### PR TITLE
#184 fix for small displays.

### LINKED ISSUE #184: Auth View - scroll window not working
see: https://forum.faforever.com/topic/6716/cant-scroll-down-the-page-when-trying-acces-downlord-s-faf-client-acess-client-page

### PR DESCRIPTION
**Pull request by fcaps**
Tested in the latest chrome/FF/Edge on Windows.
Closes #184 
faf-background

### CODE REVIEW

File: src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt [hunk 1]
```diff
@@ -26,8 +26,8 @@ class OAuthLayout(oAuthHeader: OAuthHeader) : VerticalLayout(), RouterLayout {
     }
 
     init {
-        setHeightFull()
-        setWidthFull()
+        isPadding = false
+        isSpacing = false
```
Comment (Sheikah45) [inline src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt:30]: You can just have it extend CompactVerticalLayout instead of VerticalLayout

Comment (fcaps) [reply src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt]: didn't notice we already had custom base layouts for that.

Comment (fcaps) [reply src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt]: somehow it's not working with the CompactVerticalLayout, not a kotlin-dev, so would be nice if you could tell me what's going on here.
is the init of the parent not called, or set to the wrong layout?

Comment (Sheikah45) [reply src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt]: The no-arg constructor of the super class should be called whenever the child object is created.

Is the object still having the padding and spacing classes applied when you run it locally?

Comment (fcaps) [reply src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt]: yeah.. thought it was some local cache, but it seams to have some other origin.

Comment (fcaps) [reply src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt]: got a new hint, spacing is gone, but padding is applied. i'll debug and look where this comes from

Comment (fcaps) [reply src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt]: @Sheikah45 local issue, it works.
did somehow change the content of CompactVerticalLayout, but was not aware of it.

**Pull request merged**